src/views/landing.njk

```diff
@@ -0,0 +1,83 @@
+{% extends "layout.njk" %}
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"   import govukBackLink %}
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/button/macro.njk"      import govukButton %}
+
+{% block pageTitle %}
+  {% if serviceName %}
```

Comment:
I suggest we remove that if statement and set service name globally

Reply:
The service name is set in the layout currently, I agree that if statement is not necessary